Pyflake lint fails

```
Run flake8 . --show-source --statistics
./test/test_commands.py:432:12: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
    assert type(output) == bytes
           ^
1     E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -12,11 +12,12 @@ jobs:
     strategy:
       matrix:
         cfg:
-          - {os: ubuntu-18.04, python-version: "3.6"}
           - {os: ubuntu-20.04, python-version: "3.7"}
           - {os: ubuntu-20.04, python-version: "3.8"}
           - {os: ubuntu-20.04, python-version: "3.9"}
           - {os: ubuntu-20.04, python-version: "3.10"}
+          - {os: ubuntu-22.04, python-version: "3.11"}
+          - {os: ubuntu-22.04, python-version: "3.12"}
 
     steps:
       - uses: actions/checkout@v3

--- a/README.md
+++ b/README.md
@@ -18,7 +18,7 @@ are:
 
 ## Python support
 
-This package supports python >= 3.5.
+This package supports Python >= 3.5, and is tested on Python (3.7 -> 3.12).
 
 ## How does it work?
 

--- a/test/test_commands.py
+++ b/test/test_commands.py
@@ -429,7 +429,7 @@ def run_command(command, args=None, subfolder=None):
 
 
 def adapt_command_output(output, cwd=None):
-    assert type(output) == bytes
+    assert type(output) is bytes
     # replace message from older git versions
     output = output.replace(
         b'git checkout -b new_branch_name',

